Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -44,7 +44,8 @@ bool DiscScrubber::SetupScrub(const Volume* disc, int block_size)
 
   m_file_size = m_disc->GetSize();
 
-  const size_t num_clusters = static_cast<size_t>(m_file_size / CLUSTER_SIZE);
+  // Divide m_file_size by CLUSTER_SIZE rounding up
```

Comment:
Would you mind documenting _why_ instead of the _what_? (which can be seen from the code itself)